SPECS/platform-manageability-agent/platform-manageability-agent.spec

```diff
@@ -0,0 +1,82 @@
+Summary:        Platform managebility agent for out of band opration. 
+Name:           platform-manageability-agent
+Version:        0.1.0
+Release:        1%{?dist}
+License:        Apache-2.0
+Vendor:         Intel Corporation
+Distribution:   Edge Microvisor Toolkit
+URL:            https://github.com/open-edge-platform/edge-node-agents
+Source0:        %{url}/archive/refs/tags/%{name}/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Source1:        %{name}.conf
+Source2:        %{name}.service
+Source3:        env_wrapper.sh
+Source4:        %{name}.sudoers
+BuildRequires:  golang >= 1.24.1
+BuildRequires:  systemd-rpm-macros
+Requires(pre):  %{_bindir}/systemd-sysusers
+Requires:       dmidecode
+Requires:       rpc
+
+%global debug_package   %{nil}
+%global _build_id_links none
```

Comment:
are these really required? these usually mean that we are using `-s -w` in makefile.